src/engraving/dom/textedit.cpp

```diff
@@ -142,6 +143,12 @@ void TextBase::endEdit(EditData& ed)
 
     ted->cursor()->endEdit();
 
+    if (isDynamic()) {
+        Dynamic* d = toDynamic(this);
+        const DynamicType dt = d->parseDynamicText(xmlText());
+        undoChangeProperty(Pid::DYNAMIC_TYPE, dt);
```

Comment:
Is there a reason we can't do something like this?

```suggestion
        d->setDynamicType(xmlText());
```

Reply:
My understanding is that I need to call `undoChangeProperty` to set the dynamic type consistently across all parts. Otherwise, for example, if you add or edit the dynamics in a part score, the playback will not reflect those changes.

However, while calling `undoChangeProperty` from within `Dynamic::setDynamicType` seems fine at first glance, the program crashes when I copy and paste the dynamic. This led me to believe that something is wrong, so I refrained from doing so.